src/translations/translations_NO.json

```diff
@@ -270,6 +271,7 @@
     },
     "radio_information": "Svar på spørsmålene for å få hjelp til å velge lisens eller velg lisens direkte fra nedtrekksmenyen",
     "read_more": "Les mer om ",
+    "recommended_license_is": "Anbefalt lisens basert på dine valg i dette skjemaet er {{license}}. NB! du har overstyrt valget",
```

Comment:
Kanskje ikke "du" her, ettersom det kan peke på hva som tidligere er gjort - hva om kurator er inne for å redigere ?

Reply:
Fikset